scripts/sct_image.py

```diff
@@ -127,12 +130,29 @@ def get_parser():
         help='Multi-component merge: Merge inputted images into one multi-component image. Requires several inputs.',
         required=False)
 
-    warping = parser.add_argument_group('WARPING FIELD OPERATIONSWarping field operations:')
+    warping = parser.add_argument_group('WARPING FIELD OPERATIONS:')
     warping.add_argument(
         '-display-warp',
         action='store_true',
         help='Create a grid and deform it using provided warping field.',
         required=False)
+    multi.add_argument(
+        '-world2vox',
+        metavar=Metavar.file,
+        help='Transform displacement field values from world to voxel coordinate system. Input the file that will be '
+             'used as the input (source) by FSL applywarp function.',
+        required=False)
+    multi.add_argument(
+        '-to-fsl',
+        metavar=Metavar.file,
+        help='Transform displacement field values to absolute FSL warps. To be used with FSL\'s applywarp function with the '
+        '`--abs` flag. Input the file that will be used as the input (source) for applywarp and optionally the target '
+        '(ref). The target file is necessary for the case where the warp is in a different space than the target. For '
+        'example, the inverse warps generated by `sct_straighten_spinalcord`. This feature has not been extensively '
+        'validated so consider checking the results of `applywarp` against `sct_apply_transfo` before using in FSL '
+        'pipelines',
```

Comment:
It took me some time to understand what to input and where, and then I forgot to set the flag -o so my input warp was overwritten ;-). so I suggest joining an example syntax:

```suggestion
        'validated so consider checking the results of `applywarp` against `sct_apply_transfo` before using in FSL '
        'pipelines. Example syntax: "sct_image -i WARP_SRC2DEST -to-fsl IM_SRC (IM_DEST) -o WARP_FSL", '
        'followed by FSL: "applywarp -i IM_SRC -r IM_DEST -w WARP_FSL --abs -o IM_SRC2DEST" ',
```